### PR TITLE
Response code fix

### PR DESCRIPTION
**Pull request by blueturtle85**
Fixed response code for post APIs (Category and Product)
Resolved issue: https://github.com/yhod/jello-node/issues/7

### CODE REVIEW

File: database/config/config.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
         'database': 'simple_app_db',
         'host': '127.0.0.1',
         'dialect': 'postgres',
-        logging: false,
+        logging: true,
```
Comment (yhod) [inline database/config/config.js]: Probably should not go in with this PR

File: controllers/product.controller.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ exports.create = async (req, res) => {
     try {
         const {name, description, price, category, color} = req.body;
         const product = await Product.create({name, description, price, category, color});
-        res.send(product);
+        res.status(201).send(product);
```
Comment (yhod) [inline controllers/product.controller.js:17]: You will need to change the test as well 
https://github.com/yhod/jello-node/blob/main/api/__tests/product.route.test.js#L25

File: controllers/category.controller.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ exports.create = async (req, res) => {
     try {
         const {name, description} = req.body;
         const category = await Category.create({name, description});
-        res.send(category);
+        res.status(201).send(category);
```
Comment (yhod) [inline controllers/category.controller.js:17]: Good opportunity to add a Category route test

**Review by yhod — CHANGES_REQUESTED** [review]
Please also add a link of the issue to the PR description

File: database/config/config.js (hunk 1 above)
Comment (blueturtle85) [reply database/config/config.js]: Reverted the changes.

File: controllers/product.controller.js (hunk 2 above)
Comment (blueturtle85) [reply controllers/product.controller.js:17]: Done

File: controllers/category.controller.js (hunk 3 above)
Comment (blueturtle85) [reply controllers/category.controller.js:17]: Done

**Review by yhod — APPROVED** [review]
LGTM

**Pull request merged**